lib/matplotlib/tests/test_style.py

```diff
@@ -160,3 +162,27 @@ def test_xkcd_cm():
     with plt.xkcd():
         assert mpl.rcParams["path.sketch"] == (1, 100, 2)
     assert mpl.rcParams["path.sketch"] is None
+
+
+@check_figures_equal(extensions=["png"])
+def test_seaborn_style(fig_test, fig_ref):
+    sns = pytest.importorskip('seaborn')
+
+    def make_plot(fig):
+        ax1 = fig.add_subplot(121)
+        x = np.linspace(0, 14, 100)
+        for i in range(1, 7):
+            ax1.plot(x, np.sin(x + i * .5) * (7 - i))
+
+        ax2 = fig.add_subplot(122)
+        x = np.tile(np.linspace(0, 1, 20), 2).reshape(-1, 2)
+        x[:, 1] /= 2
+        x[0, 1] = 0.8
+        ax2.boxplot(x)
+
+    plt.style.use('seaborn')
+    make_plot(fig_test)
+
+    mpl.rcParams.update(mpl.rcParamsDefault)
+    sns.set()
+    make_plot(fig_ref)
```

Comment:
The test is failing due to a limits issue and the colors of the boxplot. Does anybody have an idea what's causing this?

expected:
![grafik](https://user-images.githubusercontent.com/2836374/54479420-c0a78900-481c-11e9-8472-edfe59c601b3.png)

actual:
![grafik](https://user-images.githubusercontent.com/2836374/54479397-8a6a0980-481c-11e9-8f88-5d6c082138c9.png)

Reply:
I think the line `mpl.rcParams.update(mpl.rcParamsDefault)` should also appear before creating `fig_test`, else some old parameters, which are set by the testing environment are used.